service/apps/api.go

```diff
@@ -16,8 +16,12 @@ import (
 
 type AppsInterface interface {
 
-	// WaitGetAppIdle repeatedly calls [AppsAPI.Get] and waits to reach IDLE state
-	WaitGetAppIdle(ctx context.Context, name string,
```

Comment:
This should get reflected in https://github.com/databricks/databricks-sdk-go/blob/main/experimental/mocks/service/apps/mock_apps_interface.go#L40